src/fal/cli/fal_runner.py

```diff
@@ -46,9 +46,7 @@ def fal_run(
         )
 
 
-def _create_fal_dbt(
-    args: argparse.Namespace,
-):
+def _create_fal_dbt(args: argparse.Namespace, args_dict: Dict[str, Any]):
```

Comment:
`args` and `args_dict` both contain args, right? Can we just use one?

Reply:
I guess we could. Does not feel part of this PR though. Can we do it in another one?